themes/climate.json

```diff
@@ -29,7 +29,6 @@
   "--link--hover--color": "#837ac3",
   "--article-list-item-tags--tag-wrap--display": "none",
   "--enform-extra-header-placeholder--counter-block--progress-container--display": "none",
-  "--skewed-overlay--display": "none",
```

Comment:
I wasn't sure about these 🤔 But since we are planning to phase out campaign themes anyway, I guess it doesn't really matter...